gulp/util/webpack-config.js

```diff
@@ -56,7 +56,6 @@ const EXTERNALS = {
     "jquery": "$",
     "moment": "moment",
     "react": "React",
-    "react-addons-css-transition-group": "React.addons.CSSTransitionGroup",
```

Comment:
I've added `"react-transition-group": "react-transition-group"` - hopefully this will work! 🤞